src/core/libmaven/mzSample.cpp

```diff
@@ -155,7 +160,16 @@ void mzSample::loadSample(const char *filename)
 {
 
 	//Loading and Decoding the file
-	loadAnySample(filename);
+    //catch any error while parsing
+    try {
```

Comment:
@rish9511 Where's the case where no exception has been caught but loading sample failed?

Reply:
@sahil21 That won't happen. If for any reason uploading of sample fails, we will know about it,
either here or in [mzFileIo](https://github.com/ElucidataInc/ElMaven/pull/652/files/f9ccbce4192ae66e4d95b23e1044e4df20a6211b#diff-92d3a1d8ef6524a4f786409ff4f221c5R480)

Reply:
@rish9511 Okay. got it